gridstatus/decorators.py

```diff
@@ -56,7 +56,7 @@ def wrapped_f(*args, **kwargs):
                 save_to = args_dict.pop("save_to")
                 os.makedirs(save_to, exist_ok=True)
 
-            error = "ignore"
+            error = "raise"
```

Comment:
Is this change problematic? Without this the `ValueError` I added if date/end combos are invalid gets swallowed.

Reply:
swallowed in what situation? 

the point of this parameter is so the the caller of `get_real_time_as_market_results` can override this setting.

Reply:
> swallowed in what situation?
> 
> the point of this parameter is so the the caller of `get_real_time_as_market_results` can override this setting.

NVM. I wasn't following the logic after this line correctly.